src/future/components/Tag.js

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+
+import { RenderContext } from '@/src/future/context';
+
+const Tag = React.forwardRef(function Tag(
+  { _tag: RenderTag, _key, _meta, ...props },
+  ref
+) {
+  const renderContext = useContext(RenderContext);
+  if (!renderContext.renderTag) {
```

Comment:
Nice. Hooray for Context 🎉 Hooray for Hooks 🎉

Reply:
Glad you like it. 😄 This is actually the part I'm most nervous about, wondering if we should just keep Formatic out of the business of rendering components entirely. Where I'm at now: I think it's nice to have Formatic work out of the box, even if most consumers will tend to use hooks and provide their own presentational components. It's also good to have "reference components" to exercise the implementation.

It's _really hard_ to come up with a good model for overriding the rendering of components though, and this is about as good as good as I think it gets. There's some extraneous wrapping here, but trying to avoid the wrapping starts making things a little weirder, like with the current version where you have to worry about `...children`.